src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -50,6 +67,37 @@ const IncidentSplitContainer = ({ FormComponent }) => {
     [departments, setDirectingDepartment]
   );
 
+  const handleCompletedFormSubmit = useCallback(
```

Comment:
Long names are difficult to read especially for component internal members. How about `submitCompleted`?